<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UART TRANSMITTER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateComBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendInput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendDataButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Recived Data</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
